contracts/interfaces/IDefaultInterestRateStrategy.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import {IReserveInterestRateStrategy} from './IReserveInterestRateStrategy.sol';
+import {IPoolAddressesProvider} from './IPoolAddressesProvider.sol';
+
+/**
+ * @title IDefaultInterestRateStrategy
+ * @author Aave
+ * @notice Default Interface for the calculation of the interest rates
+ */
+interface IDefaultInterestRateStrategy is IReserveInterestRateStrategy {
+  /**
```

Comment:
Redo the docs of these functions with proper @param tags and changing the @dev to @notice, for consistency